apps/damap-frontend/src/themes/custom-theme.scss

```diff
@@ -169,6 +174,11 @@ html {
   --mat-icon-color: darkorange;
 }
 
+.material-icon-white-background-warning {
+  --mat-icon-color: white;
+  background-color: darkorange;
```

Comment:
This would be a good ooportunity to create a new warning variable in `styles.csss`, to have the warning color fixed in one place.